src/applications/claims-status/components/StemAskVAQuestions.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+function StemAskVAQuestions() {
+  return (
+    <div>
+      <h2 className="help-heading">Need help?</h2>
+      <h4>Ask Questions</h4>
+      <p>
+        <a href="https://gibill.custhelp.va.gov/app/">Ask a question online</a>{' '}
+        (Include your full name and VA file number)
+      </p>
+      <h4>Call us</h4>
+      <p>Veterans Crisis Line: </p>
+      <p>
+        <a href="tel:+1-800-273-8255">1-800-273-8255 </a>
```

Comment:
If you like, you can use the `<Telephone />` component here, as well as for the numbers below - https://design.va.gov/storybook/?path=/docs/components-telephone--default

Reply:
Hi @micahchiang, I noticed the `<Telephone />` component when working on `StemAskVAQuestions.jsx`, but saw that it drops the `1-` from the link. So in an effort to align as closely as possible to our design prototype I decided to go with the anchor tag.

Reply:
According to the [specification](https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers):

> Use +1 only when the information is specifically addressing Veterans or people who are living outside the U.S.

Please verify this with the designer.

You can customize the content of the `Telephone` component if you must, but either way it is preferred to use it, or include an `aria-label` in the link so screenreaders can properly read out the number.

Reply:
I pushed a commit which replaced all of the phone number `<a>` tags with `Telephone` components to bring those in alignment with the specification.